frontend/packages/console-shared/src/components/editor/YAMLEditorToolbar.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import ShortcutsLink from './ShortcutsLink';
+import './YAMLEditorToolbar.scss';
+
+interface YAMLEditorToolbarProps {
+  showShortcuts?: boolean;
+  toolbarLinks?: React.ReactNodeArray;
+}
+
+const YAMLEditorToolbar: React.FC<YAMLEditorToolbarProps> = ({ showShortcuts, toolbarLinks }) => {
+  if (!showShortcuts && !toolbarLinks.length) return null;
+
+  return (
+    <div className="ocs-yaml-editor-toolbar__links">
+      {showShortcuts && (
+        <div className="ocs-yaml-editor-toolbar__link">
+          <ShortcutsLink />
+        </div>
+      )}
+      {toolbarLinks &&
+        toolbarLinks.map((link, index) => (
+          <>
```

Comment:
This requires a key otherwise react will give a warning.

Reply:
Added.